SparkPost/2web2ui
# AC-1392: Subaccount assignment on Data Privacy page

### What Changed
 - Hide subaccount assignment fields when account has no subaccounts

### How To Test

1. Create a free account
1. Click "Data and Privacy" from account dropdown
1. Confirm "Subaccount Assignment" is **not** visible
1. Fill-out form
1. Click submit
1. Confirm request was successful
1. Repeat for "Multiple Recipient" tab
1. Create a subaccount
1. Click "Data and Privacy" from account dropdown
1. Confirm "Subaccount Assignment" is visible
1. Fill-out form
1. Click submit
1. Confirm request was successful
1. Repeat for "Multiple Recipient" tab
